packages/core/src/internal/reconciliation/helpers/reconcile-strategy.ts

```diff
@@ -67,7 +67,7 @@ export function reconcileStrategy(
     );
   }
 
-  const storedStrategyConfig = exState.strategyConfig;
+  const storedStrategyConfig = exState.strategyConfig ?? {};
```

Comment:
Let's add a comment explaining it here. We may accidentally delete it, because it may look unnecesary.

Reply:
see my suggestion